metrics.go

```diff
@@ -95,6 +115,33 @@ func (this *ConsumerMetrics) activeWorkers() metrics.Counter {
 	return this.activeWorkersCounter
 }
 
+func (this *ConsumerMetrics) numFetchedMessages() metrics.Counter {
+	return this.numFetchedMessagesCounter
+}
+
+func (this *ConsumerMetrics) numConsumedMessages() metrics.Counter {
+	return this.numConsumedMessagesCounter
+}
+
+func (this *ConsumerMetrics) numAcks() metrics.Counter {
+	return this.numAcksCounter
+}
+
+func (this *ConsumerMetrics) topicAndPartitionLag(topic string, partition int32) metrics.Gauge {
```

Comment:
right so we should have it more like this and write for each function what it is doing for the go doc, right?